components/sup/src/manager/service/spec.rs

```diff
@@ -164,6 +164,7 @@ impl ServiceSpec {
     pub fn default_for(ident: PackageIdent) -> Self {
         let mut spec = Self::default();
         spec.ident = ident;
+        spec.bldr_url = habitat_sup_protocol::DEFAULT_BLDR_URL.to_string();
         spec
```

Comment:
You can use this idiom to avoid using `mut`:

```rust
Self {ident: ident,
      bldr_url: DEFAULT_BLDR_URL.to_string(),
      ..Default::default()}
```